R/histogramDS.o.R

```diff
@@ -30,10 +37,9 @@ histogramDS.o <- function (xvect, num.breaks, method.indicator, k, noise){
 
   if (method.indicator==1){
 
-    # generate a number based on the input variable that will be used as a fixed random number
-    # generator
-    x.seed <- seedDS.o(xvect)
-    set.seed(x.seed)
+    # the study-specific seed for random number generation
+    seed <- getOption("datashield.seed")
```

Comment:
What is your strategy when this option is missing (opal version can be <2.14)

Reply:
Hi Yannick. Do you think that it would cause other issues if we say that the 5.0 version of datashield depends on the 2.14 version of opal?

Reply:
@davraam You can say there is a dependence on an opal version but you have no way to enforce it.